### DISCUSSION TITLE
I am getting Connection reset error

### DISCUSSION THREAD

**Discussion by pineapples5972** [plausible/analytics#4201] [Self-Hosted Support] [answered]
I am using podman. And after podman compose up -d everything seems fine in podman ps however when I head to localhost:8000 it shows error connection reset

**Comment by ruslandoga** [comment]
👋 @pineapples5972 

Please share your configuration files. Compose config, related env files, etc.

**Reply by pineapples5972** [reply, marked as answer]
It turns out it wasnt taking old database So I tried with empty database this time and it fixed it. Thanks tho.
